uk.co.compucorp.civicrm.hrsampledata/resources/csv/civicrm_activity.csv

```diff
@@ -1,46 +1,6 @@
 id,source_record_id,activity_type_id,subject,activity_date_time,duration,details,status_id,priority_id,source_contact_id,assignee_id,target_id
-1,,Public Holiday,New Year's Day,2016-01-01 00:00:00,,,Scheduled,Normal,229,,
```

Comment:
Some sample contacts are required to be removed according to the QA .. so I removed them and their related records